### PR TITLE
Adding tests for the modules decorator and utils.

### PR DESCRIPTION
**Pull request by alexandre**
WIP.

### CODE REVIEW

File: sqlalchemy_test_cache/utils.py [hunk 1]
```diff
@@ -79,6 +64,60 @@ def render_literal_value(self, value, type_):
...
+            else:
+                yield data
+
+
+def write_dump_data_to_file(dump_file_path, dump_data):
+    with open(dump_file_path, 'w') as f:
+        f.write(dump_data)
+
+
+def render_query(statement, dialect=None, reindent=True):
```
Comment (tomascorrea) [inline sqlalchemy_test_cache/utils.py]: This is a useful function but it should not be here.

File: tests/unit/test_utils.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        dump_path = utils.generate_dump_path('ClassName', 123456789)
+
+        basedir = tempfile.gettempdir()
+
+        self.assertEqual('{}/ClassName-123456789.dump'.format(basedir), dump_path)
+
+    def test_exception_when_basedir_is_not_none_but_use_tmp_is_true(self):
+
+        with self.assertRaises(ValueError) as cm:
+            utils.generate_dump_path('ClassName', 123456789, use_tmp=True, basedir='/home/alexandre')
```
Comment (tomascorrea) [inline tests/unit/test_utils.py]: /home/alexandre ?

Comment (alexandre) [inline tests/unit/test_utils.py]: Just test data, but I will update it.

File: sqlalchemy_test_cache/utils.py (hunk 1 above)
Comment (alexandre) [inline sqlalchemy_test_cache/utils.py]: Do you think we should move it to a [tiny] package?

**Pull request merged**